docs/Using-Fleet/configuration-files/README.md

```diff
@@ -823,6 +823,52 @@ You can verify that your agent options are valid by using [the fleetctl apply co
 
 Existing options will be overwritten by the application of this file.
 
+##### `command_line_flags` option
+
+**Note:** This feature requires Orbit, the Fleet agent manager: https://fleetdm.com/announcements/introducing-orbit-your-fleet-agent-manager
+
+The `command_line_flags` key inside of `agent_options` allows you to remotely manage the osquery command line flags. These command line flags are options which typically requires osquery to restart for it to take effect. But now with Orbit, one can use the `command_line_flags` key to take care of that. Orbit, will write these to the flagfile on the host, and pass it to osquery. 
+
+To see the full list of these osquery command line flags, please run `osquery` with the `--help` switch.
+
+Just like the other `agent_options` as above, remove the dashed lines (`--`) for Fleet to successfully update them.
+
+Here is an example of using the `command_line_flags` key:
+
+```yaml
+apiVersion: v1
+kind: config
+spec:
+  agent_options:
+    command_line_flags: # requires Fleet's osquery installer
+      verbose: true
+      disable_watchdog: false
+      logger_path: /path/to/logger
+```
+
+Note that the `command_line_flags` key does not support the `overrides` key, which is documented below.
+
+You can verfiy that these flags have taken effect on the hosts, by running a query against the `osquery_flags` table.
+
+If, at some point, you revoked an old enroll secret, this feature won't work for hosts that were added to Fleet using this old enroll secret. This is because Orbit uses the enroll secret to receive new flags from Fleet. For these hosts, all existing features will work as expected.
```

Comment:
So if I'm understanding this correctly, if anytime I want to rotate enroll secrets, I need to do these reinstallation steps? Seems crazy and inconvenient. Shouldn't orbit know what the enroll secret is of osquery?

Reply:
More tactically, I think some kind of header here would be useful to break out this section.

Reply:
This is kinda expected behavior. The host needs some secret to communicate with Fleet, if this secret is revoked, as expected, it won't be able to authenticate with Fleet. To update the secret on the host with a new secret, some out-of-band way is needed, regenerating the packages is one, or pushing the secret file via chef/puppet/etc is another way.

> Shouldn't orbit know what the enroll secret is of osquery?

Orbit and osquery share the same enroll secret...but how does that matter here?

Reply:
Never mind... I misread this.